src/app/games/lasca/tests/lasca.component.spec.ts

```diff
@@ -256,16 +256,46 @@ describe('LascaComponent', () => {
         }));
     });
     describe('experience as second player (reversed board)', () => {
+        it('should have first player on top', fakeAsync(async() => {
+            // Given a board that has been reversed
+            testUtils.getGameComponent().setPointOfView(Player.ONE);
+
+            // When displaying it it
+            // We need to force the updateBoard to trigger the redrawing of the board
+            await testUtils.getGameComponent().updateBoard(false);
+            testUtils.detectChanges();
+
+            // Then the square at (2, 2) should be coord (4, 4)
+            testUtils.expectElementToExist('#square_at_2_2 #square_4_4');
+            testUtils.expectElementNotToExist('#square_at_2_2 #square_2_2');
+        }));
+        it('should not duplicate highlight when doing incorrect second click', fakeAsync(async() => {
+            // Given a board where you are player two and a moving piece has been selected
+            await testUtils.expectClickSuccess('#coord_2_4');
+            const move: LascaMove = LascaMove.fromStep(new Coord(2, 4), new Coord(1, 3)).get();
+            await testUtils.expectMoveSuccess('#coord_1_3', move); // First move is set
+            await testUtils.getWrapper().setRole(Player.ONE); // changing role
+            await testUtils.expectClickSuccess('#coord_0_2'); // Making the first click
+
+            // When clicking on a invalid landing piece
+            await testUtils.expectClickFailure('#coord_0_1', LascaFailure.CAPTURE_STEPS_MUST_BE_DOUBLE_DIAGONAL());
+
+            // Then the highlight should be at the expected place only, not at their symmetric point
+            testUtils.expectElementToHaveClass('#square_0_2', 'selectable-fill');
+            testUtils.expectElementNotToHaveClass('#square_6_4', 'selectable-fill');
+        }));
         it('should show lastMove reversed', fakeAsync(async() => {
-            // Given a board on which it's player.one's turn
+            // Given a board on which it is player one's turn
```

Comment:
from player one's point of view, surtout!

Reply:
yeup, mieux